config/packages/knp_paginator.yaml

```diff
@@ -0,0 +1,10 @@
+knp_paginator:
+    page_range: 5                      # default page range used in pagination control
+    default_options:
+        page_name: page                # page query parameter name
+        sort_field_name: sort          # sort field query parameter name; to disable sorting set this field to ~ (null)
+        sort_direction_name: direction # sort direction query parameter name
+        distinct: true                 # ensure distinct results, useful when ORM queries are using GROUP BY statements
+    template:
+        pagination: "@KnpPaginator/Pagination/twitter_bootstrap_v4_pagination.html.twig"    # sliding pagination controls template
```

Comment:
If we're still on Bootstrap 3, this should be v3 I think.

Reply:
Good catch! Fixed that in commit https://github.com/sfu-dhil/pi/pull/29/commits/34e826ef8d47720073b666c4d5c57fcb24cf0000